api/routes/admin.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
Can you move this into webhooks.js? It feels like it's serving a very similar purpose of allowing us to trigger the system somehow so I'd like to keep these functional items together.

Reply:
Yes, makes sense to have this code there.